src/status_im/contexts/wallet/connected_dapps/view.cljs

```diff
@@ -1,29 +1,128 @@
   (:require
     [quo.core :as quo]
+    [quo.foundations.colors :as colors]
+    [quo.theme :as quo.theme]
     [react-native.core :as rn]
+    [react-native.safe-area :as safe-area]
+    [status-im.common.plus-button.view :as plus-button]
+    [status-im.common.resources :as resources]
+    [status-im.contexts.wallet.connected-dapps.disconnect-dapp.view :as disconnect-dapp]
     [status-im.contexts.wallet.connected-dapps.style :as style]
+    [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
+(defn- on-disconnect
+  [wallet-account {:keys [name topic]}]
+  (rf/dispatch [:hide-bottom-sheet])
+  (rf/dispatch
+   [:wallet-connect/disconnect-dapp
+    {:topic      topic
+     :on-success (fn []
+                   (rf/dispatch [:wallet-connect/remove-pairing-by-topic topic])
+                   (rf/dispatch [:toasts/upsert
+                                 {:id   :dapp-disconnect-success
+                                  :type :positive
+                                  :text (i18n/label :t/disconnect-dapp-success
+                                                    {:dapp    name
+                                                     :account (:name wallet-account)})}]))
+     :on-fail    (fn []
+                   (rf/dispatch [:toasts/upsert
+                                 {:id   :dapp-disconnect-failure
+                                  :type :negative
+                                  :text (i18n/label :t/disconnect-dapp-fail
+                                                    {:dapp    name
+                                                     :account (:name wallet-account)})}]))}]))
+
+(defn- on-dapp-disconnect-press
+  [wallet-account dapp]
+  (rf/dispatch [:show-bottom-sheet
+                {:content (fn [] [disconnect-dapp/view
+                                  {:wallet-account wallet-account
+                                   :dapp           dapp
+                                   :on-disconnect  #(on-disconnect wallet-account dapp)}])}]))
+
+(defn- account-details
+  [{:keys [name emoji color]}]
+  (let [theme (quo.theme/use-theme)]
+    [rn/view {:style style/account-details-wrapper}
+     [quo/context-tag
+      {:theme               theme
+       :type                :account
+       :size                24
+       :account-name        name
+       :emoji               emoji
+       :customization-color color}]]))
+
 (defn- header
-  [{:keys [title subtitle]}]
-  [:<>
-   [rn/view {:style style/header-container}
-    [quo/button
-     {:icon-only?          true
-      :type                :grey
-      :background          :blur
-      :size                32
-      :accessibility-label :close-scan-qr-code
-      :on-press            #(rf/dispatch [:navigate-back])}
-     :i/close]]
-   [quo/text
-    {:size   :heading-1
-     :weight :semi-bold
-     :style  (style/header-text (when subtitle true))}
-    title]])
+  [{:keys [title wallet-account on-close on-add]}]
+  (let [{:keys [color]} wallet-account]
```

Comment:
same here, if only the color is needed, then we should just pass it instead of passing the whole account map

Reply:
`wallet-account` is being passed to `account-details` component (line 80)